packages/backend/migrations/20230731141948_create_fee_table.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('fees', function (table) {
+    table.uuid('id').primary()
+    table.uuid('assetId').references('assets.id').notNullable()
+    table.enum('type', ['SENDING', 'RECEIVING']).notNullable()
+    table.timestamp('createdAt').defaultTo(knex.fn.now())
+    table.bigInteger('fixedFee')
+    table
+      .decimal('percentageFee', 5, 4)
+      .checkBetween([0, 1], 'fees_percentagefee_check')
+  })
```

Comment:
Takes values like 0.05, 0.01, etc with precision to 4 decimal places. Eg, we can represent something like 6.25% but not 6.259%.

The constraint ensures its between 0 and 1 (inclusive). Not sure if this is necessary but my thinking was it would protect against submitting something like 2.5 (thinking its 2.5%) or negatives.